easybuild/easyconfigs/r/Restrander/Restrander-20230713-GCCcore-12.2.0.eb

```diff
@@ -0,0 +1,37 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
+easyblock = 'MakeCp'
+
+name = 'Restrander'
+# No release or tag, so using the date of the last commit
+version = '20230713'
+github_account = 'mritchielab'
+local_commit = '54a61fa'
+
+homepage = "https://github.com/mritchielab/restrander"
+description = "A fast, accurate program for orienting and quality-checking cDNA sequencing reads."
+
+toolchain = {'name': 'GCCcore', 'version': '12.2.0'}
+
+source_urls = ['https://github.com/%(github_account)s/%(name)s/archive/']
+sources = [{'download_filename': '%s.tar.gz' % local_commit, 'filename': SOURCE_TAR_GZ}]
+checksums = ['8b23c9992f81896b54d73c05b447f9002fe9482d848de16e1622e09961196237']
+
+builddependencies = [
+    ('binutils', '2.39'),
+]
+
+files_to_copy = [
+    (["restrander"], "bin/"),
+    'README.md',
+    'config',
+    'figures',
+    'LICENSE',
+]
+
+sanity_check_paths = {
+    'files': ['bin/restrander', 'README.md'],
+    'dirs': ['config', 'figures'],
+}
+
```

Comment:
can you add a `sanity_check_commands` here? for example
```
sanity_check_commands = ['restrander --help']
```

Reply:
Done.